tests/integration/utils.py

```diff
@@ -275,3 +276,47 @@ def assert_content_manifest(client, request_id, image_contents):
     response_data = content_manifest_response.data
     assert_content_manifest_schema(response_data)
     assert image_contents == content_manifest_response.data["image_contents"]
+
+
+def package_manager_common_checks(test_env, env_data, tmpdir):
+    client = Client(test_env["api_url"], test_env["api_auth_type"], test_env.get("timeout"))
+    initial_response = client.create_new_request(
+        payload={"repo": env_data["repo"], "ref": env_data["ref"], "pkg_managers": env_data["pkg_manager"]}
+    )
+    completed_response = client.wait_for_complete_request(initial_response)
+    assert completed_response.status == 200
+    assert completed_response.data["state"] == "complete"
+    assert completed_response.data["state_reason"] == "Completed successfully"
+
+    expected_package_params = [env_data["package"]]
+    assert_element_from_response(completed_response.data, expected_package_params, "packages")
+    expected_deps = env_data["dependencies"]
+    assert_element_from_response(completed_response.data, expected_deps, "dependencies")
+
+    # Download and extract source tarball
+    source_name = tmpdir.join(f"download_{str(completed_response.id)}")
+    file_name_tar = tmpdir.join(f"download_{str(completed_response.id)}.tar.gz")
+    resp = client.download_bundle(completed_response.id, file_name_tar)
+    assert resp.status == 200
+    assert tarfile.is_tarfile(file_name_tar)
+    with tarfile.open(file_name_tar, "r:gz") as tar:
+        tar.extractall(source_name)
```

Comment:
This could be another function.  I think @ejegrova also had a function for this one.